exporters/geneva/include/opentelemetry/exporters/geneva/metrics/macros.h

```diff
@@ -0,0 +1,54 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <mutex>
+
+#ifndef TOKENPASTE
```

Comment:
Is this originated from somewhere or created from scratch?

Reply:
It's taken from cpp_client_telemetry github project, which is Apache2 licensed.